docker/intel-dgpu-driver.Dockerfile

```diff
@@ -19,6 +19,14 @@ RUN git clone -b RHEL86_23WW6.5_555_6469.0.3_221221.3 --single-branch https://gi
 RUN git clone -b 23WW06.5_555 --single-branch https://github.com/intel-gpu/intel-gpu-firmware.git && cd intel-gpu-firmware && mkdir -p firmware/license/ && cp -r COPYRIGHT firmware/license/
 
 FROM registry.redhat.io/ubi8/ubi-minimal
+
+LABEL vendor='Intel®'
+LABEL version='devel'
+LABEL release='1'
```

Comment:
What does release mean here?

Reply:
Release is a number used to identify the specific build for this image. In our case 1st build.
The details for meta data requirements are here - https://access.redhat.com/documentation/en-us/red_hat_software_certification/8.58/html-single/red_hat_openshift_software_certification_policy_guide/index#con-image-metadata-requirements_openshift-sw-cert-policy-container-images

Reply:
Do we need to make this version and release maps with the name patten mentioned in
https://github.com/intel/intel-data-center-gpu-driver-for-openshift/tree/main/release#general ?

Reply:
According to this link, how about we keep it simple as - 
release='550' or '555' depending on the version we are using. @uMartinXu

Reply:
You told me a release is a number used to identify the specific build. While according to https://github.com/intel/intel-data-center-gpu-driver-for-openshift/tree/main/docker#intel-data-center-gpu-driver-dockerfile-introduction
555/550 is Intel GPU driver version.

Reply:
It depends on us as we are the providers of this driver container image, and this is just a metadata for the image. So we can do release='1' and version='555' how about this?

Reply:
I am asking whether we need to map this with name patten mentioned in
https://github.com/intel/intel-data-center-gpu-driver-for-openshift/tree/main/release#general ?

Reply:
What does "just a metadata for the image" mean here? Can you educate me on what is the metadata used for here? Thanks!

Reply:
Image metadata is a mandatory requirement by RedHat for certification of any image that is going to be published on the RedHat catalog. Detailed justification is here - https://access.redhat.com/documentation/en-us/red_hat_software_certification/8.58/html-single/red_hat_openshift_software_certification_policy_guide/index#con-image-metadata-requirements_openshift-sw-cert-policy-container-images

Reply:
Then, again, do you think the "version" and "release" in the metadata should align with the name pattern in https://github.com/intel/intel-data-center-gpu-driver-for-openshift/tree/main/release#general ?

Reply:
We can map it with the release Readme documentation in the following way - 
```
name = RepositoryName: "intel-data-center-gpu-driver-container"
version = DriverVersion: "0.1.0"
release = KernelVersion: "4.18.0-372.40.1.el8_6.x86_64"
```

Reply:
This looks great. :-)